CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing
+
+Thank you for taking the time to contribute to our organization and this repository. 
+
+Please [Reference Tendermint](https://github.com/tendermint/tendermint/blob/master/CONTRIBUTING.md) for the documentation on how to contribute. 
```

Comment:
The tendermint contributing guidline is a bit too complex for this repo. Maybe it is still a good idea to link to this but somehow stating that this just serves as a good general guidline although there are differences in the projects? Does that make any sense?

Reply:
Perhaps rephrasing this a bit better https://github.com/jTendermint/jabci/wiki/Contribute :)

and basic Rust instructions from
https://github.com/tendermint/kms#development ?

Reply:
In the call I was against the dichotomy of develop and master but it seems like publishing the crate depends on pushing to master:
https://github.com/tendermint/rust-abci/blob/3f51194e80aecf5af520655637e84476af9cebd1/.circleci/config.yml#L37

I'll change the base branch of this PR to develop then.